.travis.yml

```diff
@@ -12,8 +12,8 @@ jdk:
 android:
   components:
     - tools
-    - build-tools-23.0.2
-    - android-23
+    - build-tools-28.0.2
```

Comment:
This should be 27.0.3 to match the sdk target 27. Same applies to the other modules that have been changed to 28.

Reply:
i think there is no problem with this. i am running the latest version of android studio and it says : 

> The specified Android SDK Build Tools version (27.0.3) is ignored, as it is below the minimum supported version (28.0.2) for Android Gradle Plugin 3.2.0-rc03.

if there was any problem with this android studio wouldn't suggest to use this version of build tools with my current sdk version

Reply:
i think there is no problem with this. i am running the latest version of android studio and it says :

> The specified Android SDK Build Tools version (27.0.3) is ignored, as it is below the minimum supported version (28.0.2) for Android Gradle Plugin 3.2.0-rc03.

if there was any problem with this android studio wouldn't suggest to use this version of build tools with my current sdk version

Reply:
Ah that's because the gradle plugin is set to 3.2.x which is still pre release. Set the plugin back to 3.1.0 as well. There's not need for the lib to be on beta/alpha dependencies.

Reply:
done
https://github.com/ToxicBakery/ViewPagerTransforms/pull/75/commits/8e99c32c539228a7581624bd623a9ddac2c1800d

Reply:
forgot to change .travis.yml . done in this commit: https://github.com/ToxicBakery/ViewPagerTransforms/pull/75/commits/eeb1abf187dd882852982eff2a7cdd7f6e81db8d